0.5/articles/unit-testing-elements.md

```diff
@@ -205,7 +205,7 @@ In this tutorial, we're going to add an existing Polymer element, [`<core-select
 
 First, get `<seed-element>` setup in a new `development` directory (if you haven't already in the Quick Start):
 
-    # Create a new directory to work in
+    # Create a new directory to work in (?)
```

Comment:
Sure, you can remove the '(?)'...I wasn't intending this to be merged 'as is'. I added 'markers' as I read through so I can see where to make comments, since github isn't really set up for this sort of review.